gooddata-sdk/gooddata_sdk/catalog/workspace/model/workspace.py

```diff
@@ -6,24 +6,44 @@
 from gooddata_metadata_client.model.json_api_workspace_in import JsonApiWorkspaceIn
 from gooddata_metadata_client.model.json_api_workspace_in_attributes import JsonApiWorkspaceInAttributes
 from gooddata_metadata_client.model.json_api_workspace_in_document import JsonApiWorkspaceInDocument
+from gooddata_metadata_client.model.json_api_workspace_in_relationships import JsonApiWorkspaceInRelationships
+from gooddata_metadata_client.model.json_api_workspace_in_relationships_parent import (
+    JsonApiWorkspaceInRelationshipsParent,
+)
+from gooddata_metadata_client.model.json_api_workspace_to_one_linkage import JsonApiWorkspaceToOneLinkage
 from gooddata_sdk.catalog.entity import CatalogNameEntity
 
 
 class CatalogWorkspace(CatalogNameEntity):
+    def __init__(self, workspace_id: str, name: str, parent_id: str = None):
```

Comment:
Missing return type, i.e. `-> None`

Reply:
Hmmm...it looks like mypy is no longer require typing `__init__` return type, see https://mypy.readthedocs.io/en/stable/class_basics.html#annotating-init-methods.
Taking my comment back even if it is more clear to type it, it is not required so lets leave it as it is.